### ISSUE TITLE
Opening video in a new window = error

### ISSUE THREAD

**Issue by PikachuEXE** [FreeTubeApp/FreeTube#1636]
**Behavior of the program**
Opening a new window via middle mouse button on a video would cause the new window to throw an error

**Expected behavior**
New window should play the video clicked without error

**To Reproduce**
Please add all steps to reproduce the behavior:
1. Go to 'Trending'
2. Click on video with middle mouse button
3. Wait a few seconds to ensure it's loaded
4. See blank screen with error in console

**Link to video**
Any video would reproduce

**Screenshots**
![image](https://user-images.githubusercontent.com/1018543/132093407-f2aaf831-de07-4e6c-b336-1db78a478de8.png)

**Environment Information (please complete the following information):**
 - OS: Windows
 - OS Version: Windows 10
 - FreeTube version: https://github.com/FreeTubeApp/FreeTube/actions/runs/1200562579
 - Installation Method (If applicable): Portable
 - Primary API Used: Local API

**Additional context**
Error message:
```
renderer.js:2 Uncaught ReferenceError: require is not defined
    at renderer.js:2
    at renderer.js:2
    at renderer.js:2
```

**Comment by PikachuEXE** [comment]
Same issue exists in https://github.com/FreeTubeApp/FreeTube/releases/tag/v0.14.0-beta

**Comment by kettenfett** [comment]
Can confirm on Windows 10 with portable FreeTube 0.14 beta.

This is an important feature for me, because I like to open multiple video-windows as a sort of queue to watch in a session.

Please please, fix this bug. Thanks.

**Comment by azurusnova** [comment]
I can confirm this as well on my Linux Mint 20.2 install. 
It's a feature that I widely use a lot as well.

**Comment by jasonhenriquez** [comment]
Seems this problem is a subset of #1652.

**Comment by PrestonN** [comment]
I think #1652 is actually a duplicate of this, so I'll go and close it. This is the same issue.

**Comment by anedroid** [comment]
This issue touches Ubuntu version too, I think on all platforms. It doesn't appear in previous FreeTube versions, only on 0.14-beta. I won't be able to update until this error will be fixed.

**Comment by PikachuEXE** [comment]
By testing locally via
- revert suspicious commit 
- `npm ci`
- `npm run debug`

I am able to fix the issue by reverting commit 0db6dd1f970747e34336e964245350cbfdcf7f21

**Comment by anedroid** [comment]
@PikachuEXE  Does one of modules updates break something in the app?

**Comment by PikachuEXE** [comment]
I have figured out which update breaks it yet
It's time consuming to pick each update and `npm ci` and `npm run debug`...

**Comment by RevAngel7** [comment]
I can confirm this issue on Ubuntu Linux. Middle mouse click only brings up an empty window that has the menu bar visible on top.

**Comment by stpnwf** [comment]
> I can confirm this issue on Ubuntu Linux. Middle mouse click only brings up an empty window that has the menu bar visible on top.

Yes, I get the same empty window on Ubuntu as well

**Comment by 360fun** [comment]
Same issue on Mac OS Big Sur!

**Comment by PikachuEXE** [comment]
This is fixed by #1658

**Comment by 360fun** [comment]
Working in the latest nightly! ;)
